app-modules/authorization/tests/Feature/Filament/Resources/User/ViewUserTest.php

```diff
@@ -124,30 +124,31 @@
     expect(auth()->id())->toBe($second->id);
 });
 
-it('does not display the mfa_status Action for an external User', function () {
+it('does not display the mfa_status for an external User', function () {
     $user = User::factory()->external()->create();
 
     asSuperAdmin();
 
     livewire(ViewUser::class, [
         'record' => $user->getRouteKey(),
     ])
-        ->assertActionHidden('mfa_status');
+        ->assertDontSeeText('MFA Enabled')
+        ->assertDontSeeText('MFA Enabled | Not Confirmed')
+        ->assertDontSeeText('MFA Disabled');
```

Comment:
@ankit-canyon, after giving this some more thought. What do you think about applying something like a data attribute to the MFA badge in the HTML and asserting that it exists with something like `assertSeeHtml('data-identifier="mfa-enabled"')` and like assertDontSeeHtml('data-identifier="mfa-enabled-not-confirmed"')`.

This way we are being very explicit about testing that the elemetns we expect are on the page or not on the page rather than checking whether this string exists.